bower.json

```diff
@@ -1,8 +1,8 @@
 {
-  "name": "git-task-list",
-  "dependencies": {
-    "highlightjs": "^9.10.0",
-    "showdown": "^1.6.4",
-    "localforage": "~1.3.1"
-  }
+    "name": "git-task-list",
```

Comment:
If bower.json *must* have a four space indent style, create a different section in .coafile for it.
That rule will be needed on all repos with a `bower.json`, if it is a specific need of `bower.json`

Reply:
It is not specific for bower, it is for all json files. I cant find specific rule for spacing published by bower.

Reply:
But doesnt bower default to 2 space indents?
https://github.com/bower/bower/issues/1338#issuecomment-194291721

Reply:
It is, I talked about this with @blazeu. @blazeu asked to ignore it, since i have to reformat the bear to follow your suggestion, it is not ignored yet.

Reply:
Will be in the next push